OpenOversight/app/models.py

```diff
@@ -492,6 +510,22 @@ class User(UserMixin, BaseModel):
     classifications = db.relationship("Image", backref="users")
     tags = db.relationship("Face", backref="users")
 
+    def _jwt_encode(self, payload, expiration):
+        secret = current_app.config["SECRET_KEY"]
```

Comment:
I'd like for `SECRET_KEY` to be a constant since we use it more than once.

Reply:
For me this is a variable name (presented as a string, similarly to `getattr(obj, "name")` for `obj.name`), and it seems easier to follow than having to do one additional lookup to find the value of the variable.